src/contracts/pods/EigenPod.sol

```diff
@@ -94,20 +103,25 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
 
     /// @notice Emitted when an ETH validator's  balance is proven to be updated.  Here newValidatorBalanceGwei
     //  is the validator's balance that is credited on EigenLayer.
-    event ValidatorBalanceUpdated(uint40 validatorIndex, uint64 newValidatorBalanceGwei);
+    event ValidatorBalanceUpdated(uint40 validatorIndex, uint64 timestamp, uint64 newValidatorBalanceGwei);
     
     /// @notice Emitted when an ETH validator is prove to have withdrawn from the beacon chain
-    event FullWithdrawalRedeemed(uint40 validatorIndex, address indexed recipient, uint256 withdrawalAmountWei);
+    event FullWithdrawalRedeemed(uint40 validatorIndex, uint64 timestamp, address indexed recipient, uint256 withdrawalAmountWei);
 
     /// @notice Emitted when a partial withdrawal claim is successfully redeemed
-    event PartialWithdrawalRedeemed(uint40 validatorIndex, address indexed recipient, uint64 partialWithdrawalAmountGwei);
+    event PartialWithdrawalRedeemed(uint40 validatorIndex, uint64 timestamp, address indexed recipient, uint64 partialWithdrawalAmountGwei);
```

Comment:
why are the partial withdrawal and validator balance update event amounts of type uint64 / in Gwei, while the amount for the full withdrawal is in wei? seems like a recipe for messing something up in indexing

Reply:
@gpsanant thoughts on the correct format here? I would think all uint256 / wei amounts makes the most sense?

Reply:
uint64 timestamps are fine and maybe preferred because they work better with sql and are what's used on the beacon chain

Reply:
either is fine, i have no opinion for amounts

Reply:
Ok switched to Gwei for FullWithdrawalRedeemed